File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--displaying-custom-task-panes-with-e-mail-messages-in-outlook_9.cs
        private void ThisAddIn_Shutdown(object sender, System.EventArgs e)
        {
            inspectors.NewInspector -=
                new Outlook.InspectorsEvents_NewInspectorEventHandler(
                Inspectors_NewInspector);
            inspectors = null;
            inspectorWrappersValue = null;
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--displaying-custom-task-panes-with-e-mail-messages-in-outlook_9.vb
    Private Sub ThisAddIn_Shutdown(ByVal sender As Object, ByVal e As System.EventArgs) _
        Handles Me.Shutdown

        RemoveHandler inspectors.NewInspector, AddressOf Inspectors_NewInspector
        inspectors = Nothing
        inspectorWrappersValue = Nothing
    End Sub